quickstart/2-browser-parallel-computation/src/App.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useEffect, useRef, useState } from 'react';
+import logo from './logo.svg';
+import './App.scss';
+
+import { Fluence, kras, type ConnectionState } from '@fluencelabs/js-client';
+import { resolveSubnet, add_one_parallel, add_one_sequential, add_one_single } from './_aqua/getting-started';
```

Comment:
`_aqua` doesn't seem like a suitable name for the generated js code

where does `fcli` put generated js by default? maybe let's use fcli's structure.

Reply:
that's using cli ts template:
```
$ tree src
src
├── aqua
│   └── main.aqua
└── js
    ├── package.json
    └── src
        ├── aqua
        └── index.js

5 directories, 3 files```